packages/dear-imgui/cpp/app/src/imgui_renderer.cpp

```
@@ -0,0 +1,522 @@
+    }
+
+    texture->textureView = view;
+    texture->width = width;
+    texture->height = height;
+
+    stbi_image_free(stbiData);
+
+    return true;
+}
+#else
+bool LoadTexture(const void* data, const int numBytes, Texture* texture) {
```

cppcheck

[style] The function 'LoadTexture' is never used.

(unusedFunction)